embedchain/vectordb/weaviate.py

```diff
@@ -231,23 +234,25 @@ def query(self, input_query: List[str], n_results: int, where: Dict[str, any], s
                 weaviate_where_clause = {"operator": "And", "operands": weaviate_where_operands}
 
             results = (
-                self.client.query.get(self.index_name, ["text"])
+                self.client.query.get(self.index_name, data_fields)
                 .with_where(weaviate_where_clause)
                 .with_near_vector({"vector": query_vector})
                 .with_limit(n_results)
                 .do()
             )
         else:
             results = (
-                self.client.query.get(self.index_name, ["text"])
+                self.client.query.get(self.index_name, data_fields)
                 .with_near_vector({"vector": query_vector})
                 .with_limit(n_results)
                 .do()
             )
         matched_tokens = []
         for result in results["data"]["Get"].get(self.index_name):
-            matched_tokens.append(result["text"])
-
+            context = result["text"]
+            # source = result["url"]
+            # doc_id = result["doc_id"]
```

Comment:
Are these lines intentionally commented?

Reply:
Yes, will be turned on after DB setup changes.